app/lib/Plugins/SearchEngine/Elastic8/FieldTypes/DateRange.php

```diff
@@ -82,6 +82,10 @@ function getFiltersForTerm($term, ?string $field = null): array {
 		$return = [];
 
 		$parsed_values = caGetISODates($parse_date);
+		if (!$parsed_values['start'] && !$parsed_values['end']) {
```

Comment:
If we have neither of these then we would be searching from the "begining of time - end of time"